core/domain/config_validators.py

```diff
@@ -76,7 +76,8 @@ def _get_external_id_relationships(cls, item):
                 [item.id[:item.id.rfind(base_models.VERSION_DELIMITER)]]),
             base_model_validators.ExternalModelFetcherDetails(
                 'committer_ids',
-                user_models.UserSettingsModel, [item.committer_id])]
+                user_models.UserSettingsModel, [item.committer_id], True,
+                True)]
```

Comment:
I would mention the keywords explicitely, ditto in other places
```suggestion
                user_models.UserSettingsModel, 
                [item.committer_id], 
                allow_system_user_ids=True,
                allow_pseudonymous_ids=True
            )]
```

Reply:
Done.

Reply:
Not done?

Reply:
Done.